### ISSUE TITLE
logrotate cron job reports error for influxdb

### ISSUE THREAD

**Issue by codyshepherd** [influxdata/influxdb#22435]
reported log error:

```
"/etc/cron.daily/logrotate:
error: Ignoring influxdb because it is writable by group or others."
```

Possibly related to permissions issue of logrotate files laid down by influxdb installation:
```
If I take a look inside the package, the permissions for the logrotate file are set incorrectly:
" rpm -qvl influxdb2-2.0.8-1.x86_64 | grep logrotate
-rw-rw-r--    1 root    root                      113 Aug 13 20:14 /etc/logrotate.d/influxdb"
```

Possibly new in 2.0.8 and/or 2.x

**Comment by boutetnico** [comment]
I'm also affected by this issue.
A simple temporarily fix:
```bash
chmod g-w /etc/logrotate.d/influxdb
```

**Comment by jdstrand** [comment]
The temporary fix indeed works but gets overwritten on upgrade. @bnpfeife - I think the umask must be 002 on the system this is being generated on. Can you add this to your list of the next release?

**Comment by CendioMartin** [comment]
This affects us as well on all servers where we have InfluxDB installed. It's a pain to get hit by this cron warning each time the package gets bumbed.

**Comment by bnpfeife** [comment]
This should now be fixed with #24306
